shim/build.rs

```diff
@@ -89,12 +101,12 @@ fn build_debug_libturbo() -> String {
             .success(),
         "failed to build turbo static library"
     );
-    cli_path.to_string_lossy().to_string()
+    cli_path
 }
 
-fn header_path(target: &build_target::Os) -> &'static str {
-    match target {
-        build_target::Os::Windows => "../cli/turbo.h",
-        _ => "../cli/libturbo.h",
-    }
```

Comment:
`ld` requires `libturbo.h` so getting clever here for Rust's bindings gets us literally nowhere.

Reply:
This is incredibly annoying. The windows dev story is again one filled with mystery.